### ISSUE TITLE
Not able to use python expression

### ISSUE THREAD

**Issue by aborruso** [saulpw/visidata#205]
Hi,
I have loaded datetime module and I have a column with dates. The name of this field is "data".
If I use this expression `expr=datetime.strptime(data, '%Y-%m-%d').month` I have no value.

Ho to reference the field "data" in the expression??

Thank you

**Comment by saulpw** [comment]
Hi @aborruso, use the `=` command and then type the `datetime.strptime`... expression.  Don't put `expr` at the beginning.

**Comment by aborruso** [comment]
Hi @saulpw 

> Hi @aborruso, use the = command and then type the datetime.strptime... expression. Don't put expr at the beginning.

I know but I have

![image](https://user-images.githubusercontent.com/30607/47968858-ca9bcd00-e06f-11e8-8505-0e5d3412f98d.png)

If I use a fixed value it works

![image](https://user-images.githubusercontent.com/30607/47968873-f4ed8a80-e06f-11e8-81cd-707eea3c4a88.png)

Thank you

**Comment by zakora** [comment]
Hi,

I think the error is that you are using the `datetime` module but visidata doesn't provide it as is when writting python expression.

What you can do instead:
- mark the `data` as date with `@`, as you did
- you can now use the data column directly as datetime objects, so you can start an expr with `=`, then write `data.month`.

**Comment by aborruso** [comment]
Hi @zakora ,
I can use datetime moduke because I have inserted it in `.visidatarc` file. And for this reason `datetime.strptime("2018-08-08", '%Y-%m-%d').month` works.
Than it should work also using some existing field.

But your great tip to use simply data.month, after applying @ gives me the answer.

I applied `datetime.strptime` to a field that was already datetime. Then I had to apply "~" to the field and it works great!

Thank you

![image](https://user-images.githubusercontent.com/30607/47969301-d5f1f700-e075-11e8-9c77-d7fa21eec447.png)




My `.visidatarc`

```
options.color_key_col=''
options.color_selected_row='yellow'
options.color_note_type='yellow'
options.color_graph_hidden='blue'
options.color_column_sep='blue'

from datetime import datetime
```
